cfme/infrastructure/provider/virtualcenter.py

```diff
@@ -1,5 +1,16 @@
 from mgmtsystem.virtualcenter import VMWareSystem
 from . import Provider
+from cfme.web_ui import Form, Input
+from cfme.common.provider_endpoint import ProviderEndpoint
+
+
+default_endpoint_form = Form(
```

Comment:
I wonder if this should be defined inside a `ProviderEndpointDefault` class seeing as its closely tied to it. Same for other endpoint types. So `ProviderEndpointBase` would define the generic fill and classes like `ProviderEndpointCandU` and `ProviderEndpointDefault` would contain the fields definition and would potentially also know how to parse those fields from provider config.
```
endpoints = [
    ProviderEndpointDefault.from_config(prov_config),
    ProviderEndpointCandU.from_config(prov_config)]
```
Id say thats easy to understand, no magic happening there and its easy to maintain.
Thoughts?